src/ntfs/read.rs

```diff
@@ -295,7 +295,7 @@ where
             |v| v != 0,
         );
 
-        let precision = stop_time.precision.or_else(|| {
+        let precision = stop_time.precision.or({
```

Comment:
I'm surprised by this change since there is no reason to evaluate the `if` when `stop_time.precision` is `Some`. Why not keeping it lazy?

Reply:
ask for CLippy https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_lazy_evaluations

I think it's not necessary for simple types

Reply:
Weird. Especially since this is not a simple type because there is an evaluation (an `if` in one case, a function in the other one). But hey, if `clippy` say so, since our CI depends on it, fine by me!

Reply:
wait for the pull request of `tartare-tools` before being shocked